### PR TITLE
Category code par 3215

### PR DESCRIPTION
**Pull request by ereng**
@briankip check it out

### CODE REVIEW

File: app/views/reports/patient/index.blade.php [hunk 1]
```diff
@@ -41,7 +41,7 @@
 			@forelse($patients as $key => $value)
```
Comment (briankip) [inline app/views/reports/patient/index.blade.php]: Could you rename $value to $patient its a bit confusing.

**Comment by briankip** [comment]
* Try and move specimenID next to test
* Add searching with the new specimen Ids
* see if you can add the ability to create sequential specimen Ids on a lab-section basis, @mapesa tackled this on the old blis.

**Comment by ereng** [comment]
@briankip  see if this will do.

**Comment by briankip** [comment]
@mapesa saidia please.

**Comment by mapesa** [comment]
ngumu sana - siwezi leo. Has to be tomorrow or the next day.

Like I told you, I'm under pressure to merge some of our functionality to
the C4G codebase.

**Pull request merged**